packages/react/src/internal/FeatureFlags.js

```diff
@@ -9,30 +9,40 @@
  * This file contains the list of the default values of compile-time feature flags.
  */
 
+/**
+ * This flag will determine if all feature flags should be enabled
+ *
+ * @type {boolean}
+ */
+export const DDS_FLAGS_ALL = process.env.DDS_FLAGS_ALL === 'true' || false;
+
 /**
  * Feature flag to turn on the Masthead L1
  *
  * @type {boolean}
  */
-export const MASTHEAD_L1 = process.env.MASTHEAD_L1 === 'true' || false;
+export const MASTHEAD_L1 =
+  process.env.MASTHEAD_L1 === 'true' || (DDS_FLAGS_ALL || false);
 
 /**
  * This determines if the locale selector will be rendered or not
  *
  * @type {string | boolean}
  */
 export const FOOTER_LOCALE_BUTTON =
-  process.env.FOOTER_LOCALE_BUTTON === 'true' || false;
+  process.env.FOOTER_LOCALE_BUTTON === 'true' || (DDS_FLAGS_ALL || false);
 
 /**
  * This flag turns on/off the ButtonGroup component
  *
  * @type {string | boolean}
  */
-export const BUTTON_GROUP = process.env.BUTTON_GROUP === 'true' || false;
+export const BUTTON_GROUP =
+  process.env.BUTTON_GROUP === 'true' || (DDS_FLAGS_ALL || false);
 
 /**
  * Feature flag for CardLink component
  * @type {boolean}
  */
-export const CARD_LINK = process.env.CARD_LINK === 'true' || false;
+export const CARD_LINK =
+  process.env.CARD_LINK === 'true' || (DDS_FLAGS_ALL || false);
```

Comment:
If we want the feature flag in a broader scope like stories, then maybe we could look at expanding the `featureFlag` utility function. I only used JSX when creating it because that's all I thought we'd be using it for at the time. We could maybe move to a callback approach.

```javascript
function featureFlag(flag, cb) {
  if (flag) {
    if (typeof cb === 'function') {
      return cb(flag)
    } else {
      return cb;
    }
  }

  return null;
}
```